Mubashir/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>College Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Bakbak+One&family=Inter:wght@100..900&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="navbar">
+   <div class="moreicon">
+    <img src="More.png">
+   </div>
+   <div class="updates">
+    <h1>UPDATES</h1>
+   </div>
+</div>
+
+<div class="content">
+    <p class="randomtext">Lorem tem ullam tenetur perspiciatis ipsam enim.
+    Magnam dolori isnxnsnxnxxk tfssvcd ugsv xuhsvsabbvsabs sbbhhbbbbi</p>
+    <p class="messages">A new unread messages!</p>
+    <p class="projects">Projects!</p>
+    <p class="generalmsg">ALL CAUGHT UP!</p>
```

Comment:
instead of <p> make use of the <div> with a unique id so that it would be easy to add styles and manipulate that element using the js

Reply:
can't understand bro 
would you plz eloberate....